regression/cbmc-primitives/r_w_ok_inconsistent_integer/main.c

```diff
@@ -0,0 +1,11 @@
+#include <assert.h>
+#include <stdlib.h>
+
+void main()
+{
+  // pointer with object bits = 123 and offset = 123
```

Comment:
Thanks for the improvement - as the reader, I'm still left with the question of _why_ you're constructing such a pointer. I think this would still be improved with a comment as to why this is an interesting pointer. Is it because it tricks cbmc into thinking it is valid?

Reply:
Yes, it's a pointer that doesn't point to valid memory but isn't special in any other way. Added a description.